kms/api-client/asymmetric_test.py

```diff
@@ -99,44 +99,55 @@ def test_get_public_key(self):
         assert isinstance(ec_key, _EllipticCurvePublicKey), 'expected EC key'
 
     def test_rsa_encrypt_decrypt(self):
-        ciphertext = sample.encryptRSA(self.message,
-                                       self.client,
-                                       self.rsaDecrypt)
+        ciphertext_bytes = sample.encryptRSA(self.message_bytes,
+                                             self.client,
+                                             self.rsaDecrypt)
+        ciphertext = ciphertext_bytes.decode('utf-8')
         # ciphertext should be 344 characters with base64 and RSA 2048
         assert len(ciphertext) == 344, \
             'ciphertext should be 344 chars; got {}'.format(len(ciphertext))
         assert ciphertext[-2:] == '==', 'cipher text should end with =='
-        plaintext = sample.decryptRSA(ciphertext, self.client, self.rsaDecrypt)
+        plaintext_bytes = sample.decryptRSA(ciphertext_bytes,
+                                            self.client,
+                                            self.rsaDecrypt)
+        assert plaintext_bytes == self.message_bytes
+        plaintext = plaintext_bytes.decode('utf-8')
         assert plaintext == self.message
 
     def test_rsa_sign_verify(self):
-        sig = sample.signAsymmetric(self.message, self.client, self.rsaSign)
+        sig = sample.signAsymmetric(self.message_bytes,
+                                    self.client,
+                                    self.rsaSign)
         # ciphertext should be 344 characters with base64 and RSA 2048
         assert len(sig) == 344, \
             'sig should be 344 chars; got {}'.format(len(sig))
         assert sig[-2:] == '==', 'sig should end with =='
         success = sample.verifySignatureRSA(sig,
-                                            self.message,
+                                            self.message_bytes,
                                             self.client,
                                             self.rsaSign)
         assert success is True, 'RSA verification failed'
+        changed_bytes = (self.message+".").encode('utf-8')
```

Comment:
Nit: alternatively, `changed+bytes = message_bytes + b'.'`.

Reply:
That does look nicer. Changed